src/views/catalog/templatescatalog/components/TemplatesCatalogDrawer/TemplatesCatalogDrawerCreateForm.tsx

```diff
@@ -29,173 +29,170 @@ import {
+                        />
+                      </FormGroup>
+                    </SplitItem>
+                    <SplitItem>
+                      <DescriptionList>
+                        <DescriptionListGroup>
+                          <DescriptionListTerm>{t('Project')}</DescriptionListTerm>
+                          <DescriptionListDescription>{namespace}</DescriptionListDescription>
+                        </DescriptionListGroup>
+                      </DescriptionList>
+                    </SplitItem>
+                  </Split>
+                </StackItem>
+                <StackItem />
+                <StackItem>
+                  <Checkbox
+                    id="start-after-create-checkbox"
+                    isChecked={startVM}
+                    onChange={(v) => setStartVM(v)}
+                    label={t('Start this VirtualMachine after creation')}
+                  />
+                </StackItem>
+              </>
+            ) : (
+              <StackItem>
+                {t(
+                  'This Template requires some additional parameters. Click the Customize VirtualMachine button to complete the creation flow.',
+                )}
               </StackItem>
-              <StackItem />
+            )}
+            <StackItem />
+            {quickCreateError && (
               <StackItem>
-                <Checkbox
-                  id="start-after-create-checkbox"
-                  isChecked={startVM}
-                  onChange={(v) => setStartVM(v)}
-                  label={t('Start this VirtualMachine after creation')}
-                />
+                <Alert variant="danger" title={t('Quick create error')} isInline>
+                  {quickCreateError?.message}
+                </Alert>
               </StackItem>
-            </>
-          ) : (
-            <StackItem>
-              {t(
-                'This Template requires some additional parameters. Click the Customize VirtualMachine button to complete the creation flow.',
-              )}
-            </StackItem>
-          )}
-          <StackItem />
-          {quickCreateError && (
-            <StackItem>
-              <Alert variant="danger" title={t('Quick create error')} isInline>
-                {quickCreateError.message}
-              </Alert>
-            </StackItem>
-          )}
+            )}
 
-          <StackItem>
-            <Split hasGutter>
-              {canQuickCreate && (
+            <StackItem>
+              <Split hasGutter>
+                {canQuickCreate && (
+                  <SplitItem>
+                    <Button
+                      data-test-id="quick-create-vm-btn"
+                      type="submit"
+                      form="quick-create-form"
+                      isLoading={isQuickCreating || modelsLoading}
+                      isDisabled={
+                        !isBootSourceAvailable || isQuickCreating || !vmName || modelsLoading
```

Comment:
I think it would be better to use isEmpty(models) instead of modelsLoading, wdyt?

Reply:
Yes, we can do that.